<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Person</class>
 <widget class="QDialog" name="Person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>физ.лицо</string>
  </property>
  <widget class="QTableView" name="person_table_vew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>971</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="first_name_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="middle_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="inn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_batch">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="city">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="street">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="house">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="flat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_number">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="last_name_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="middle_name_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="inn_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>инн</string>
   </property>
  </widget>
  <widget class="QLabel" name="pass_batch_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>серия паспорта</string>
   </property>
  </widget>
  <widget class="QLabel" name="city_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>город</string>
   </property>
  </widget>
  <widget class="QLabel" name="street_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>улица</string>
   </property>
  </widget>
  <widget class="QLabel" name="house_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>53</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>дом</string>
   </property>
  </widget>
  <widget class="QLabel" name="flat_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>квартира</string>
   </property>
  </widget>
  <widget class="QLabel" name="pass_number_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>номер паспорта</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
